matrix-guide.md

```diff
@@ -0,0 +1,58 @@
+# Matrix new user guide for TC39 delegates
+
+1. Choose a client
+
+    Like IRC and unlike many closed platforms, Matrix is an open protocol. Therefore, you can choose which client(s) you would prefer to use based on your preferred workflow.
+
+    If you don't care too much and just want something that "works", we recommend [Element] (web, desktop and mobile).
+
+    If you want to explore further or experiment with alternate clients, check out this [list of clients].
+
+1. Create an account
+
+    (Skip this step if you already have a matrix account).
+
+    1. Choose a homeserver
+
+        Matrix is a federated service. Unlike IRC, you don't necessarily need to register on each server you want to talk on, you can register on any server you trust/like and talk everywhere else.
+
+        We recommend using the officially maintained matrix.org homeserver, unless you have a reason to do otherwise. Matrix.org is the default on most clients. This is the server our channels are hosted on and a majority of delegates are registered to.
+
+        Other servers are also available. For instance, Mozilla and Igalia delegates tend to use their own corporate homeservers. If you want to explore other servers, you can find many to choose from in the following lists:
+        - https://www.hello-matrix.net/public_servers.php
+        - https://publiclist.anchel.nl/
+
+    1. Create your account
+
+        Check the documentation for your preferred client. For most clients including Element, there should be a link to do this on the landing page when you're not already logged into another account.
+
+1. Log into your account and start talking
+
+    1. Log into your account
+
+        Check the documentation for your preferred client. For most clients including Element, there should be a link to do this on the landing page when you're not already logged into another account.
+
+    1. Join the official TC39 channels
+
+        You can click through the following links to join the various official TC39 channels in your preferred client. They all correspond to the similarly titled channels on Freenode.
+
+        1. [#tc39-general]
```

Comment:
consider adding descriptions of channels.  here are some from what I was sent in a welcome email:

#tc39-general:matrix.org:
 public conversation, open to community members

#tc39-delegates:matrix.org:
 publicly viewable conversation, participation restricted to delegates

#temporaldeadzone:matrix.org:
 off-topic conversations and bachkchanneling. Technically public, but intended for delegates

#tc39-inclusion:matrix.org:
 an informal working group with the goal of making TC39 a more inclusive committee in which to participate.

Reply:
sounds great, I'd be happy to add descriptions for all of them.